### PR TITLE
KAA-1137 C SDK Linux page

### PR DESCRIPTION
**Pull request by forGGe**

### CODE REVIEW

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Linux/index.md [hunk 1]
```diff
@@ -11,7 +11,10 @@ sort_idx: 10
 * TOC
 {:toc}
 
-This guide describes how to use *C SDK* on linux-based systems.
+The application code based on Kaa C SDK is similar for most of the platform used.
+However, build system is not.
+
+Throughout the guide we focus on details of the CMake-based build system that is used to tie application code together with the C SDK.
```
Comment (rasendubi) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Linux/index.md]: `Throughout the guide,`. Missed comma

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Linux/index.md [hunk 2]
```diff
@@ -11,7 +11,10 @@ sort_idx: 10
 * TOC
 {:toc}
 
-This guide describes how to use *C SDK* on linux-based systems.
+The application code based on Kaa C SDK is similar for most of the platform used.
+However, build system is not.
```
Comment (ohamov) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Linux/index.md:15]: probably should be moved to the main C SDK page with the corresponding links list to the platform-dependent build procedure descriptions

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Linux/index.md [hunk 3]
```diff
@@ -11,7 +11,10 @@ sort_idx: 10
...
+
+Throughout the guide, we focus on details of the CMake-based build system that is used to tie application code together with the C SDK.
```
Comment (ohamov) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Linux/index.md:17]: may be the word `connect` is more appropriate usage than `tie` from the technical point of view?

**Pull request merged**